mono/utils/mono-dl.c

```diff
@@ -168,6 +168,14 @@ fix_libc_name (const char *name)
 }
 #endif
 
+int
+mono_dl_flags_external_to_internal (int flags)
+{
+	if (!(flags & MONO_DL_LOCAL))
```

Comment:
What is the expected behavior for specifying local and global?

Reply:
In this case at least, it shouldn't change anything. If MONO_DL_GLOBAL is already true, this will just preserve that. I'd rather address the conflict in `mono_dl_convert_flags`.